sdk/src/connections/offline/offline_depth_sensor.cpp

```diff
@@ -24,7 +24,8 @@ OfflineDepthSensor::~OfflineDepthSensor() {
 }
 
 aditof::Status OfflineDepthSensor::getFrame(uint16_t *buffer) {
-    auto frame = m_frameTypes.find(m_frameTypeSelected + ".bin");
+    auto frame =
+        m_frameTypes.find(std::to_string(m_frameTypeSelected) + ".bin");
```

Comment:
I guess that this means you need to update the .bin files, right?

Reply:
Yes